client/packages/system/src/Patient/Encounter/CreateEncounterForm.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { FC, useState } from 'react';
```

Comment:
Pulled the shared form out `CreateEncounterModal` so can be shared between create and schedule next 🚀